src/js/api.js

```diff
@@ -1,80 +0,0 @@
-
-var development = true; 
-
-/******************************
-         api.js 
- ******************************/
-
-// Define global DV (Data Visualization) Configurations
-
-var DV = {
-	
-  // Define common CRUD functions
-  api : {},
-
-  // Is this a development configuration?
-  // Disables logging, and debug information
-  // for production environments.
-  development : development,
-
-  // Target API
-  url : (development) ? "http://localhost:8080/" : null,
-};
-
-/******************************
-      CRUD Functionality
- ******************************/
-
-// Reference on XHR callbacks
-// http://stackoverflow.com/questions/5485495/how-can-i-take-advantage-of-callback-functions-for-asynchronous-xmlhttprequest
-
-// GET
-// takes target URL, callback, callback
-DV.api.get = function(url, success, error){
```

Comment:
I was trying to reduce our reliance on jQuery. It might be a pipe dream, but I'd like to be able to drop jQuery as a dependency. I might resort back to $.get(...) when we plug the twitter API back in.